tests/integration/__tests__/__snapshots__/schema.js.snap

```diff
@@ -89,6 +89,11 @@ exports[`schema 1`] = `
             },
           ],
         },
+        "experimentalOperatorPosition": {
+          "default": false,
+          "description": "When binary expressions wrap lines, put operators at the start of new lines.",
+          "type": "boolean",
+        },
```

Comment:
This can't be a boolean, It should be `"line-start" "line-end"` or maybe `"start" | "end"` ?

Or maybe change name to `experimentalOperatorLinebreak`, like [`operator-linebreak`](https://eslint.org/docs/latest/rules/operator-linebreak) rule.

@sosukesuzuki I forgot how we decided.

Reply:
@fisker check out 039802c03 – I went with `experimentalOperatorPosition: start|end` since that's the less verbose version. Despite the ESLint rule precedent, I personally find it easier to think about where to place the operator rather than the linebreak relative to the operator. If you all have a strong preference for some other form, I'm happy to tweak it.

Reply:
I'm fine with current name.

Reply:
would be awesome to get this merged!